GeneratorInterface/Herwig7Interface/test/DYToLL01234Jets_5FS_TuneCH3_13TeV_madgraphMLM_herwig7_cfg.py

```diff
@@ -0,0 +1,264 @@
+        'create ThePEG::Cuts /Herwig/Cuts/NoCuts', 
+        'cd /Herwig/EventHandlers', 
+        'create Herwig::FxFxFileReader FxFxLHReader', 
+        'insert LesHouchesHandler:FxFxReaders[0] FxFxLHReader', 
+        'cd /Herwig/Shower', 
+        'library HwFxFxHandler.so', 
+        'create Herwig::FxFxHandler FxFxHandler', 
+        'set /Herwig/Shower/FxFxHandler:SplittingGenerator /Herwig/Shower/SplittingGenerator', 
+        'set /Herwig/Shower/FxFxHandler:KinematicsReconstructor /Herwig/Shower/KinematicsReconstructor', 
+        'set /Herwig/Shower/FxFxHandler:PartnerFinder /Herwig/Shower/PartnerFinder', 
+        'set /Herwig/EventHandlers/LesHouchesHandler:CascadeHandler /Herwig/Shower/FxFxHandler', 
+        'set /Herwig/Partons/PDFSet_nnlo:PDFName NNPDF31_nnlo_as_0118', 
+        'set /Herwig/Partons/RemnantDecayer:AllowTop Yes', 
+        'set /Herwig/Partons/PDFSet_nnlo:RemnantHandler /Herwig/Partons/HadronRemnants', 
+        'set /Herwig/Particles/p+:PDF /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/Particles/pbar-:PDF /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/Partons/PPExtractor:FirstPDF  /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/Partons/PPExtractor:SecondPDF /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/Shower/ShowerHandler:PDFA /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/Shower/ShowerHandler:PDFB /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/EventHandlers/FxFxLHReader:FileName cmsgrid_final.lhe', 
+        'set /Herwig/EventHandlers/FxFxLHReader:WeightWarnings false', 
+        'set /Herwig/EventHandlers/FxFxLHReader:AllowedToReOpen No', 
+        'set /Herwig/EventHandlers/FxFxLHReader:InitPDFs 0', 
+        'set /Herwig/EventHandlers/FxFxLHReader:Cuts /Herwig/Cuts/NoCuts', 
+        'set /Herwig/EventHandlers/FxFxLHReader:MomentumTreatment RescaleEnergy', 
+        'set /Herwig/EventHandlers/FxFxLHReader:PDFA /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/EventHandlers/FxFxLHReader:PDFB /Herwig/Partons/PDFSet_nnlo', 
+        'set /Herwig/Shower/ShowerHandler:MaxPtIsMuF Yes', 
+        'set /Herwig/Shower/ShowerHandler:RestrictPhasespace Yes', 
+        'set /Herwig/Shower/PartnerFinder:PartnerMethod Random', 
+        'set /Herwig/Shower/PartnerFinder:ScaleChoice Partner', 
+        'set /Herwig/Shower/KinematicsReconstructor:InitialInitialBoostOption LongTransBoost', 
+        'set /Herwig/Shower/KinematicsReconstructor:ReconstructionOption General', 
+        'set /Herwig/Shower/KinematicsReconstructor:InitialStateReconOption Rapidity', 
+        'set /Herwig/Shower/ShowerHandler:SpinCorrelations Yes', 
+        'cd /Herwig/Shower', 
+        'set /Herwig/Shower/FxFxHandler:MPIHandler  /Herwig/UnderlyingEvent/MPIHandler', 
+        'set /Herwig/Shower/FxFxHandler:RemDecayer  /Herwig/Partons/RemnantDecayer', 
+        'set /Herwig/Shower/FxFxHandler:ShowerAlpha  AlphaQCD', 
+        'set FxFxHandler:HeavyQVeto Yes', 
```

Comment:
@Dominic-Stafford : Did you check how to apply the setting?

Reply:
Andreas said that this setting should be set to "Yes" if we want to match the b jets to b quarks, and recommended we do this- apparently this is independent of whether we use the 4fs or the 5fs